release.json

```diff
@@ -10,6 +10,17 @@
 		"OMNIBUS_RUBY_BRANCH": "datadog-5.5.0",
 		"OMNIBUS_SOFTWARE_BRANCH": "master"
 	},
+	"datadog-vsphere-rc.1": {
```

Comment:
Can we remove this release entry from the PR? Just use an auxiliary branch to trigger the build from jenkins.